cimr/processor/processor_prompt.py

```diff
@@ -149,7 +149,16 @@ def processor_cli(args):
 
         if args.yaml_file:
             yaml_file = [pathlib.Path(args.yaml_file),]
-            y, genome_build, fileset, columnset = convert_yaml(yaml_file)
+            y, fileset = convert_yaml(yaml_file)
+
+            # Check Yamler for required arguments
+            if hasattr(y, 'columnset'):
+                columnset = y.columnset
+            else:
+                columnset = {}
+            if hasattr(y, 'genome_build'):
```

Comment:
Don't you need a default value for `genoe_build` too (just as you did with `columnset`)? Without the default value, if `y.genome_build` doesn't exist, line 176 would complain that the variable is not defined.

Reply:
submission without genome_build should have exited at `convert_yaml()`

Reply:
So lines 159-160 can be simply written into:
```
genome_build = y.genome_build
```
right?

Reply:
sure